doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -39,6 +44,28 @@ dashbuilder/navigation/readme.md
 VERSION
 ----
 
+
+If you want to export customized user created dashbuilder data as a ZIP file, do the following tasks:
+
+.. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+.. In *Export Wizard* panel, select the datasets which you want to be present in the ZIP file and click *Next*.
+.. Select the Pages which you want to be present in the ZIP file and click *Next*.
++
+The *Export Wizard* panel validates the selected datasets and pages. The summary of the datasets and pages is available on the panel.
++
+NOTE: By default, Navigation is always exported.
+
+.. Click *Download* if your export is ready.
++
+An `export.zip` file containing customized dashbuilder data is downloaded.
++
+NOTE: You must select the pages and the datasets associated with those pages in the *Export Wizard* panel. You must select at least one page. If you fail to select both datasets and pages, an export error is generated and you can not download the `export.zip` file.
+
+.. Click *Finish*.
+.. Optional: Click *Cancel* to return to the *Dashbuilder Data Transfer* page.
```

Comment:
Question - The cancel implies that a process is cancelled but this step is at the end of the procedure after the file has been downloaded.

Suggestion - remove this step?